src/current/v23.2/cutover-replication.md

```diff
@@ -148,7 +152,30 @@ To monitor for when the replication stream completes, use [`SHOW VIRTUAL CLUSTER
     SET CLUSTER SETTING server.controller.default_target_cluster='application';
     ~~~
 
-At this point, the primary and standby clusters are entirely independent. You will need to use your own network load balancers, DNS servers, or other network configuration to direct application traffic to the standby (now primary). To enable physical cluster replication again, from the new primary to the original primary (or a completely different cluster), refer to [Cut back to the primary cluster](#cut-back-to-the-primary-cluster).
+At this point, the primary and standby clusters are entirely independent. You will need to use your own network load balancers, DNS servers, or other network configuration to direct application traffic to the standby (now primary). To manage replicated jobs on the promoted standby, refer to [Job management](#job-management).
+
+To enable physical cluster replication again, from the new primary to the original primary (or a completely different cluster), refer to [Cut back to the primary cluster](#cut-back-to-the-primary-cluster).
+
+## Job management
+
+During a replication stream, jobs running on the primary cluster will replicate to the standby cluster. Once you have [completed a cutover](#step-2-complete-the-cutover) (or a [cutback](#cut-back-to-the-primary-cluster)), refer to the following sections for details on resuming jobs on the promoted cluster.
+
+### Backup schedules
+
+[Backup schedules]({% link {{ page.version.version }}/manage-a-backup-schedule.md %}) will pause after cutover on the promoted cluster. Take the following steps to resume jobs:
+
+1. Verify that there are no other schedules running backups to the same [collection of backups]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#backup-collections), i.e., the schedule that was running on the original primary cluster.
+1. Resume the backup schedule on the promoted cluster.
+
+{{site.data.alerts.callout_info}}
+If your backup schedule was created on a cluster in v23.1 or earlier, it will **not** pause automatically on the promoted cluster after cutover. In this case, you must pause the schedule manually on the promoted cluster and then take the outlined steps.
+{{site.data.alerts.end}}
+
+### Changefeeds
+
+[Changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}) will fail on the promoted cluster immediately after cutover. We recommend that you recreate changefeeds on the promoted cluster.
+
+[Scheduled changefeeds]({% link {{ page.version.version }}/create-schedule-for-changefeed.md %}) will continue on the promoted cluster. You will need to manage [pausing]({% link {{ page.version.version }}/pause-schedules.md %}) or [canceling]({% link {{ page.version.version }}/drop-schedules.md %}) the schedule on the original primary and promoted standby clusters.
```

Comment:
@msbutler I don't know if this is quite correct. I made an assumption of what would happen here because scheduled changefeeds are a one-time table scan rather than a continuous job like a regular changefeed. Please correct me! 
Also, I have not added this as a limitation yet, do we want to do so?

Reply:
I think what you have here is fine! Perhaps you could explain why we recommend some manual intervention: we don't recommend two clusters writing changefeeds to the same sink. 

We should definitely add a known limitation for this.